### DISCUSSION TITLE
Static<typeof Model> vs typeof Model.static

### DISCUSSION THREAD

**Discussion by pauldvu** [sinclairzx81/typebox#603] [Q&A] [answered]
Is there a major difference or any drawbacks to inferring types like in example 1?
```typescript
//example 1
 type T = typeof Model.static;
```

```typescript
//example 2
import { Static  } from "@sinclair/typebox";

type T = Static<typeof Model>
```

I quite like doing typeof Model.static, and wasn't sure if there is a downside to inferring types like that. It also saves an extra import , and is useful for quickly inning the type definition. 

In zod I would do something like
```typescript
type T = typeof Schema._type

```
as opposed to 

```typescript
type T = z.infer<typeof Schema>

```

**Comment by sinclairzx81** [comment, marked as answer]
@pauldvu Hi,

You should be able to use `typeof T.static` or `Static<typeof T>` interchangeable. You're free to use either form, but be mindful that the `.static` property is a phantom property, and the Static<T> usage does provide an abstraction over that detail.

Hope that helps!
S

**Reply by pauldvu** [reply]
Appreciate it @sinclairzx81 ,  I had to read a few blog posts to learn about phantom types! 😅  This is good to know!
